src/TwigComponent/tests/Fixtures/templates/components/Alert.html.twig

```diff
@@ -0,0 +1,26 @@
+{% props color = 'blue', size = 'md' %}
+
+{% set alert = cva('alert', {
```

Comment:
So the first argument is still a thing? What does it do?

Reply:
https://github.com/symfony/ux/pull/1416#discussion_r1498352255
Add some bases classes 😁

Reply:
Oh alright, I didn't see it thanks. 

Do you think this really needs to be required?

Reply:
It's not mandatory you can have an empty value, it's just we don't have named arguments in Twig, but none of those arguments should be mandatory